Src/Plugins/Tests.Security/Validators/SEC101_047.CratesApiKeyValidatorTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Helpers;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    public class CratesApiKeyValidatorTests
+    {
+        [Fact]
+        public void DiscordCredentialsValidator_MockHttpTests()
```

Comment:
>DiscordCredentialsValidator_MockHttpTests [](http://example.com/codeflow?start=20&length=41)

Why did you remove that other test? It was a good one.

Reply:
It wasn't testing anything.
It was a blank test when I want to test a specific fingerprint.